modules/sql/src/main/resources/db/migration/V056__Fits_KeywordNames.sql

```diff
@@ -0,0 +1,663 @@
+
+---------------------------
+-- FITS Keyword Names
+---------------------------
+
+CREATE TABLE e_fits_keyword_names (
+    id   identifier       PRIMARY KEY,
+    name VARCHAR(8)       NOT NULL CHECK (name  ~     '[A-Z0-9_-]{1,8}')
+);
+
+ALTER TABLE e_fits_keyword_names OWNER TO postgres;
+
+COPY e_fits_keyword_names (id, name) FROM stdin;
+INSTRUMENT	INSTRUME
```

Comment:
Is it possible to have to fill the second column only when the keyword is different from the id ?

Reply:
I think you’d need to add a NULL at the end of every other row